Formula/qemu.rb

```diff
@@ -85,6 +84,10 @@ def install
       ["--enable-gtk"]
     end
 
+    # The --enable-slirp flag (not the feature itself) was removed in the head:
+    # https://gitlab.com/qemu-project/qemu/-/commit/5890258aeeba303704ec1adca415e46067800777
+    args += ["--enable-slirp=system"] if !build.head?
```

Comment:
```suggestion
    args << "--enable-slirp=system" if build.stable?
```
Please squash this change into your existing commit. Thanks!

Reply:
Thanks, done